README.md

```diff
@@ -1,52 +1,75 @@
-# CSI Snapshotter
+# CSI Snapshot Feature
```

Comment:
Can you also mirror the README format that the other sidecars have?

Reply:
With Beta, I'll have to talk about the snapshot controller and the CSI snapshotter.  So it won't be exactly the same as other sidecars.